src/legacy/core_plugins/interpreter/common/types/error.ts

```diff
@@ -17,26 +17,27 @@
  * under the License.
  */
 
-export const pointseries = () => ({
-  name: 'pointseries',
-  from: {
-    null: () => {
-      return {
-        type: 'pointseries',
-        rows: [],
-        columns: {},
-      };
-    },
-  },
+import { ExpressionType } from '../../types';
+import { Render } from './render';
+
+const name = 'error';
+
+export interface InterpreterError {
+  type: typeof name;
+  error: unknown;
+  info: unknown;
```

Comment:
I'm not sure if these are accurate typings for errors.

Also wondering if we should use another name besides `InterpreterError`, which may eventually be used for actual errors thrown by the interpreter. Maybe `InterpreterErrorType` or just `ErrorType`?

Reply:
I would add a `// TODO` here with an issue for follow-up.